deploy/helm/rawfile-localpv/values.yaml

```diff
@@ -6,6 +6,11 @@ logLevel: INFO
 # -- Format of the logs (json, pretty)
 logFormat: json
 
+# -- Reserved storage amount on each node for data dir storage on each host (Supports percentage and size) [e.g. `25%` or `50GB` or `10MiB`]
```

Comment:
hmm would something like `reservedCapacity` be a better name to keep in-line with the `capacityOverride` ?

Reply:
Done.